### DISCUSSION TITLE
Start implementing TRAPI 1.1.0-beta

### DISCUSSION THREAD

**Discussion by edeutsch** [NCATSTranslator/ReasonerAPI#228] [Show and tell]
Hi everyone, TRAPI 1.1.0-beta is now officially committed in the `v1.1.x` branch:
https://github.com/NCATSTranslator/ReasonerAPI/tree/v1.1.x
it is not yet in `master`.

The YAML is here:
https://github.com/NCATSTranslator/ReasonerAPI/blob/v1.1.x/TranslatorReasonerAPI.yaml

The ChangeLog is here:
https://github.com/NCATSTranslator/ReasonerAPI/blob/v1.1.x/ChangeLog.md

Autogenerated documentation markdowns still lag behind. We will fix that over the next coming weeks.
It seems likely that during the implementation phase we will discover something we goofed up and will want to fix before we officially tag 1.1.0, so be prepared for a small tweak.

We still also have several policy decisions on how we will use the schema under various circumstances. We will work on that over the next coming weeks as well.

The Pull Requests area is clean, everything remaining is tagged for version 1.2:
https://github.com/NCATSTranslator/ReasonerAPI/pulls

Please begin your implementations of TRAPI 1.1.0-beta and speak up about problems you encounter. Our target is to have everyone implemented TRAPI 1.1 by the May Relay meeting.

**Comment by brettasmi** [comment]
Thanks @edeutsch and everyone in the working group who contributed to this.

**Comment by edeutsch** [comment]
I relayed what I thought our TRAPI plan was at today's Deployment call when asked:
- Teams should maintain v1.0 (SmartAPI and ARS) while implementing v1.1-beta
- When a v1.1-beta endpoint is ready:
o	Register it separately from the v1.0 endpoint in SmartAPI
o	Log a ticket for Mark in Relay for ARS Dev
- ARS Dev will support a combination of v1.0 and v1.1-beta 
- TRAPI v1.0 Endpoints will be deprecated June 1

Does seem right compared with what you remember?
